### PR TITLE
CVSB-19353 - Handle large SQS messages

### PR DESCRIPTION
**Pull request by waringr**
Adds the ability to handle large SQS messages by offloading them to S3 if required. This needs to be released in conjunction with similar changes in the cvs-tsk-edh-marshaller and cvs-tsk-update-store, otherwise the large messages will be lost by the system, and probably cause an increase in errors.

### CODE REVIEW

**Review by SanjeetJ-TA — DISMISSED** [review]
Happy to approve

**Review by SanjeetJ-TA — APPROVED** [review]
Approved.

File: src/utils/sqs-huge-msg.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    if (Body !== undefined) {
+      messages[0].Body = await this.getMessageContent(Body);
+    }
+
+    await this.deleteMessage(queueName, message);
+
+    return message;
+  }
+
+  public async getMessageContent(body:string):Promise<string> {
```
Comment (waringr) [inline src/utils/sqs-huge-msg.ts:131]: Return parsed body & rename

Comment (waringr) [reply src/utils/sqs-huge-msg.ts:131]: Not required due to getMessage depending on this function and requiring a string. So this method would need wrapping in something to parse the string response and casting the output, at which point all we're doing is moving the parsing into the library and making it less flexible due to the type-casting.

**Review by m-ujaffer — APPROVED** [review]
Approving with the tech debt that `sqs-huge-msg.ts` is duplicated across `cvs-tsk-update-store`, `cvs-tsk-edh-dispatcher`, `cvs-tsk-edh-marshaller`. Suggested to convert to a package in later iterations.

**Pull request merged**